qf_lib/data_providers/bloomberg/historical_data_provider.py

```diff
@@ -18,6 +18,7 @@
 import numpy as np
 import pandas as pd
 from pandas import to_datetime
+import pandas as pd
```

Comment:
duplicate of line 19

Reply:
It's removed on line 19 (I moved the line by mistake)